oak-core/src/test/java/org/apache/jackrabbit/oak/query/stats/QueryRecorderTest.java

```diff
@@ -18,12 +18,48 @@
 
 import static org.junit.Assert.assertEquals;
 
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.TreeSet;
+
 import org.junit.Test;
 
+import com.google.common.io.LineReader;
+
 public class QueryRecorderTest {
     
+    public static void main(String... args) throws IOException {
```

Comment:
This looks an unintended change

Reply:
Well... the "/Users/mueller/temp/queryRecorder.txt" is unintended... but I'll remove the whole thing, it's better